app/assets/stylesheets/layouts/starrydark.scss

```diff
@@ -1,157 +1,287 @@
+// TODO: See if we can differentate these
+$bg_color_flash_error: #484357; // error
+$bg_color_flash_success: #484357; // success
+$bg_color_flash_inbox: #484357; // new message
+$bg_color_flash_breadcrumbs: #131937; // breadcrumbs ("Continuities » Continuity » Section » Thread")
+$font_color_flash: #DADADA;
+$font_color_breadcrumbs_bold: #F3F3F3;
+
+.flash.breadcrumbs {
+  background-color: $bg_color_flash_breadcrumbs;
+  &, a, a:visited { color: $font_color_link_visited; }
+  b { color: $font_color_breadcrumbs_bold; } // TODO: Investigate why this is differentated
 }
+.flash.success { background-color: $bg_color_flash_success; color: $font_color_flash; }
+.flash.error { background-color: $bg_color_flash_error; color: $font_color_flash; }
+.flash.inbox { background-color: $bg_color_flash_inbox; color: $font_color_flash; }
 
-$EVEN_COLOR: #31323B;
-$ODD_COLOR: #25262E;
+/* - main page */
+$bg_color_page: #211E2F; // overall page
+$font_color_page: #DADADA;
 
-#post-menu-box {
-  border-top: 1px solid #808080;
-  border-top: 1px solid rgba(128,128,128,0.5);
+body { background-color: $bg_color_page; color: $font_color_page; }
 
-  a { color: inherit; }
-  div {
-    border-bottom: 1px solid #808080;
-    border-bottom: 1px solid rgba(128,128,128,0.5);
-  }
+$bg_color_spacer_alt: #131937;
+$font_color_search_box: #FEFEFE;
+$bg_color_search_box: #131937;
+
+.search-box { background-color: $bg_color_search_box; color: $font_color_search_box; }
+.spacer-alt { background-color: $bg_color_spacer_alt }
+
+/* - misc */
+$font_color_descriptions: #DADADA;
+$bg_color_descriptions: #484357; // description subheadings on templates page ("single-description"), *not* posts
+.single-description {
+  background-color: $bg_color_descriptions;
+  color: $font_color_descriptions;
+  a { color: $font_color_link; }
+  a:visited { color: $font_color_link_visited; }
+}
+
+/* - content headers */
+$bg_color_head: #0D0B1E; // primary content header
+$bg_color_sub: #131937; // secondary content header
+$bg_color_subber: #484357; // tertiary content header
+$bg_color_ender: #0D0B1E; // form table footer with submit
+$font_color_ender: #DADADA;
+$font_color_sub: #F3F3F3;
+$font_color_subber: #DADADA;
+$font_color_subber_bold: #F3F3F3;
+
+th, .content-header {
+  background-color: $bg_color_head;
+  a { color: $font_color_link; }
+  a:visited { color: $font_color_link_visited; }
+}
+.sub { color: $font_color_sub; background-color: $bg_color_sub; }
+.form-table-ender { background-color: $bg_color_ender; color: $font_color_ender; }
+
+.subber {
+  background-color: $bg_color_subber;
+  color: $font_color_subber;
+  &, a, a:visited { color: $font_color_link_visited; }
+  b { color: $font_color_subber_bold; }
 }
+
+/* - content */
+$bg_color_even: #31323B;
+$bg_color_odd: #25262E;
+$opacity_blockquote: 0.2;
+
 .even, .post-post, .post-reply:nth-child(even), #post-editor {
-  background-color: $EVEN_COLOR;
+  background-color: $bg_color_even;
+  color: $font_color_page;
```

Comment:
This is added color, which is probably okay but not definitely; @Throne3d check that the font color hasn't changed and maybe remove this in favor of fixing starry's page color thingy? (So they consistently stay the right thing.)

Reply:
edit: I lie. see below a thing got eaten.